.travis.yml

```diff
@@ -2,17 +2,27 @@ language: python
 cache:
     directories:
         $HOME/.cache/pip
-python: 3.5
 sudo: false
 addons:
     apt:
         packages:
             - libzookeeper-mt-dev
-env:
-    - TOX_ENV=py27
-    - TOX_ENV=py35
-    - TOX_ENV=pypy
-    - TOX_ENV=flake8
+matrix:
+    include:
+        - python: 2.7
+          env: TOX_ENV=py27
+        - python: 3.3
+          env: TOX_ENV=py33
+        - python: 3.4
+          env: TOX_ENV=py34
+        - python: 3.5
+          env: TOX_ENV=py35
+        - python: 3.6
+          env: TOX_ENV=py36
+        - python: pypy
+          env: TOX_ENV=pypy
+        - python: 3.5
```

Comment:
Is this line supposed to point to `python: 3.5`?  I wasn't sure about the usage of `flake8` in a tox file.